l10n_be_cooperator/views/tax_shelter_declaration_view.xml

```diff
@@ -47,21 +47,21 @@ SPDX-License-Identifier: AGPL-3.0-or-later
                 <sheet>
                     <group>
                         <group>
+                            <field name="date_from" />
+                            <field name="date_to" />
+                            <field name="fiscal_year" widget="char" />
                             <field name="name" />
-                            <field name="fiscal_year" />
+                        </group>
+                        <group>
                             <field name="tax_shelter_type" />
                             <field name="tax_shelter_capital_limit" />
                             <field name="previously_subscribed_capital" />
                         </group>
-                        <group>
-                            <field name="date_from" />
-                            <field name="date_to" />
-                        </group>
```

Comment:
i’m not sure about this: i think that it should be possible to select the year first, which should set the date range, which can be adjusted afterwards. what rationale did you follow? the fact that a declaration is always done for the current year, except when testing?

moreover, there is no test that `date_to` is after `date_from`. this should be added to avoid mistakes.

Reply:
>moreover, there is no test that date_to is after date_from. this should be added to avoid mistakes.

There is a check that throws an error, but I need to add automatic tests, you're right